app/models/import_trade_summary.rb

```diff
@@ -0,0 +1,12 @@
+require 'api_entity'
+
+class ImportTradeSummary
+  include ApiEntity
+
+  collection_path '/import_trade_summary'
+
+  attr_accessor :basic_third_country_duty,
+                :preferential_tariff_duty,
+                :preferential_quota_duty
+
```

Comment:
```ruby

def no_preferential_duties?
  !(preferential_tariff_duty && preferential_quota_duty)  
end
```

Reply:
nice one,
I guess the logic should be
`!(preferential_tariff_duty || preferential_quota_duty) `
or (same logic):
`preferential_tariff_duty.nil? && preferential_quota_duty.nil?`